src/components/sheet/sheet-action-buttons/DepositActionButton.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { colors } from '../../../styles';
+import { neverRerender } from '../../../utils';
+import {
+  FloatingEmojis,
+  FloatingEmojisTapHandler,
+} from '../../floating-emojis';
+import SheetActionButton from './SheetActionButton';
+
+const emojis = ['soon', 'soon', 'soon', 'soon', 'unicorn', 'soon', 'rainbow'];
+
+const DepositActionButton = ({ color = colors.dark, ...props }) => (
+  <FloatingEmojis
+    distance={350}
+    duration={2000}
+    emojis={emojis}
+    size={36}
+    wiggleFactor={0}
+  >
+    {({ onNewEmoji }) => (
+      <FloatingEmojisTapHandler onNewEmoji={onNewEmoji}>
+        <SheetActionButton {...props} color={color} label="􀁍 Deposit" />
```

Comment:
We should track this

Reply:
done

Reply:
hmmmm....tbh when I discovered this button, I then proceeded to click it a LOT bc it was fun 😅 could be overkill but maybe we just have some state that only sends the analytics tracking once per mount instead (also, same thing with the Withdraw button)

Reply:
😂️ good point will do @jinchung